src/controllers/crossCutting.controller.js

```diff
@@ -0,0 +1,75 @@
+const stringifyImgPath = require("../util/stringifyImgPath");
+const editFrames = require("../services/editFrames");
+const exportFinalVideo = require("../services/exportFinalVideo");
+
+async function crossCutting(req, res, next) {
+  // const singleShots = res.locals.singleShots;
+  // const editPoints = res.locals.editPoints;
+  // const { subOneFrames, subTwoFrames } = await sortFrames(singleShots, editPoints);
+
+  // TODO: 잠시 Mock Data를 사용합니다. main 브랜치에 병합 전에 변경예정입니다.
+  const subOneFrames = {
+    1740: 3,
+    2190: 4,
+    2610: 4,
+    3690: 3,
+    4530: 2,
+  };
+  const subTwoFrames = {
+    90: 2,
+    330: 2,
+    450: 2,
+    570: 3,
+    840: 4,
+    1290: 3,
+    2910: 3,
+    3090: 4,
+    3300: 5,
+    4170: 4,
+    4350: 5,
+    6450: 2,
+  };
+
+  if (subOneFrames && subTwoFrames) {
+    const subOneFrameList = Object.entries(subOneFrames);
+    const subTwoFrameList = Object.entries(subTwoFrames);
+
+    if (subOneFrameList.length) {
+      const subOnePromiseList = subOneFrameList.map(
+        async ([frameNumber, durationTime]) => {
+          const mainFramePath = stringifyImgPath("main-contents", frameNumber);
+          const subFramePath = stringifyImgPath(
+            "sub-one-contents",
+            frameNumber,
+          );
+
+          await editFrames(mainFramePath, subFramePath, durationTime);
+        },
+      );
+
+      await Promise.all(subOnePromiseList);
+    }
+
+    if (subTwoFrameList.length) {
+      const subTwoPromiseList = subTwoFrameList.map(
+        async ([frameNumber, durationTime]) => {
+          const mainFramePath = stringifyImgPath("main-contents", frameNumber);
+          const subFramePath = stringifyImgPath(
+            "sub-two-contents",
+            frameNumber,
+          );
+
+          await editFrames(mainFramePath, subFramePath, durationTime);
+        },
+      );
+
+      await Promise.all(subTwoPromiseList);
+    }
+  }
+
+  await exportFinalVideo();
+
+  res.status(200).send("success");
```

Comment:
수진님! 저희가 result 값에 success 혹은 fail로 통일하기로 했으므로 아래와 같이 변경되어야할 것 같습니다!
메세지의 경우는 필요시 넣으면 될 것 같습니다 👍

```suggestion
  res.status(200).send({ result: "success", message: "video exported" }); 
```

Reply:
확인 후 변경하였습니다.